src/D2L.CodeStyle.Analyzers/Immutability/MutabilityAuditor.cs

```diff
@@ -3,15 +3,24 @@
 
 namespace D2L.CodeStyle.Analyzers.Immutability {
 	internal sealed class MutabilityAuditor {
-		public static bool IsAudited( ISymbol symbol, out Location location ) {
+		public static bool IsAudited( ISymbol symbol, out Location location, out Diagnostic diagnostic ) {
 			AttributeData attr;
+			diagnostic = null;
+
+			// Check if there is a conflicting audit AND unaudit
+			if( ( Attributes.Statics.Audited.IsDefined( symbol ) || Attributes.Mutability.Audited.IsDefined( symbol ) )
+				&& ( Attributes.Statics.Unaudited.IsDefined( symbol ) || Attributes.Mutability.Unaudited.IsDefined( symbol ) ) ) {
+				diagnostic = Diagnostic.Create(
+					Diagnostics.ConflictingAuditing,
+					symbol.DeclaringSyntaxReferences[0].GetSyntax().GetLastToken().GetLocation() );
+			}

```

Comment:
This will raise a diagnostic for the following code

```csharp
[Statics.Unaudited( Because.Foo )]
[Mutability.Audited( "Owen Smith", "2020-12-17", "Foo" )]
private bool m_disposed;
```

While it's still invalid code _regardless_, these issues are about trying to improve (or get back to good) the developer experience and give the correct suggestions.

We don't want `ConflictingAudit` to be raised for this case - instead this would just have a diagnostic for the `Statics.Unaudited` being used on a non-static member. That particular diagnostic doesn't need to happen in this PR though.

`ConflictingAudit` should be raised for Audit+Unaudited of the same `Statics`/`Mutability`.